federatedscope/mf/trainer/trainer.py

```diff
@@ -66,6 +82,13 @@ def _hook_on_batch_end(self, ctx):
         ctx.loss_batch_total += ctx.loss_batch.item() * ctx.batch_size
         ctx.loss_regular_total += float(ctx.get("loss_regular", 0.))
 
+        if self.cfg.federate.method.lower() in ["fedem"]:
+            # cache label for evaluation ensemble
+            ctx.get("{}_y_true".format(ctx.cur_mode)).append(
```

Comment:
The attribute `y_true` is a matrix here and can be very large for MF dataset, I'm not sure it's appropriate to storage all the labels and probs